index.js

```diff
@@ -0,0 +1,21 @@
+import { DateTime } from './node_modules/luxon/src/luxon.js';
```

Comment:
- [ ] [OPTIONAL] Instead of importing the `DateTime` from the `/node_modules` directory it would be convenient to add a `luxon.js` file with the [code found here](https://moment.github.io/luxon/es6/luxon.js) in the `modules` directory and import it like `import { DateTime } from './luxon.js'`
    - This way the date time is not dependent on the `/node_modules`. In other words, you don't have to run `npm i`. ✔